tests/Integration/UseCases/HandlePayPalPaymentNotification/HandlePayPalPaymentNotificationUseCaseTest.php

```diff
@@ -231,15 +245,15 @@ public function testGivenNewTransactionIdForBookedDonation_transactionIdShowsUpI
 			$fakeRepository,
 			new SucceedingDonationAuthorizer(),
 			$this->getMailer(),
-			new NullLogger()
+			new NullLogger(),
+			$this->getEventLogger()
```

Comment:
This can be avoided by setting default services in setUp. An yeah, I'm aware I was complaining about setUp before :)